tools/kprint/main.cpp

```diff
@@ -62,40 +102,142 @@ int main (int argc, char **argv) {
   comms.insert("\\or");
   std::map<std::string, std::vector<std::string>> colors;
 
-  // load information about definition written by k frontend
-  std::ifstream iformats(argv[1] + std::string("/format-att.txt"));
-  std::ifstream ihooks(argv[1] + std::string("/hook-att.txt"));
-  std::ifstream iassocs(argv[1] + std::string("/assoc-att.txt"));
-  std::ifstream icomms(argv[1] + std::string("/comm-att.txt"));
-  std::ifstream icolors(argv[1] + std::string("/color-att.txt"));
+  std::map<std::string, std::set<std::string>> priorities, leftAssoc, rightAssoc;
```

Comment:
What are these big tables here? Any way we can load them from the actual information in `domains.k` and `kast.k` instead?

Reply:
Maybe? it's kinda tricky though. These tables contain the metadata for the stuff that is defined in kast.k and thus doesn't get translated by ModuleToKORE, like kseq, inj, dotk, \or, \and, etc. I'm not sure how I can get this information from the frontend to the backend when there aren't symbols in the definition that I can automatically insert attributes for. I'd like to! I'm just not sure how to go about it. Do you have an idea?